docs/introduction.rst

```diff
@@ -1 +1,6 @@
+************
+Introduction
+************
+
 .. include:: ../README.rst
+   :start-line: 16
```

Comment:
It feels like there's no way we're going to remember to update this (or at least no way I'm going to remember), and then our introduction page is going to be broken until someone happens to notice.  That possibility outweighs the benefit of skipping the badges I'd say.  Maybe we could add a big note in `README.rst` saying "don't forget to go update `introduction.rst`"?  Although even then it seems like it'd be pretty easy not to notice that comment if you're touching a different part of the readme.

Reply:
The benefit is more skipping the Nengo Bones header than the badges but yeah let me try the other method of doing this to see if it can be done.